File: backend/src/main/java/com/bluebee/smartsupply/model/Shop.java
package com.bluebee.smartsupply.model;

import java.io.Serializable;

public class Shop implements Serializable {

    private Integer shopcd;
    private String shopname;
    private String ownername;
    private String addess1;
    private String address2;
    private String address3;
    private String emailid;
    private String zipcode;
    private String mobileno;
    private Integer appusercd;
    private Integer status;

    transient String itemname;

    public String getItemname() {
        return itemname;
    }

    public void setItemname(String itemname) {
        this.itemname = itemname;
    }

    public Integer getShopcd(){
    return this.shopcd;
    }
    public void setShopcd(Integer shopcd){ 
    this.shopcd=shopcd;
    }
    public String getShopname(){ 
    return this.shopname;
    }
    public void setShopname(String shopname){ 
    this.shopname=shopname;
    }
    public String getOwnername(){ 
    return this.ownername;
    }
    public void setOwnername(String ownername){ 
    this.ownername=ownername;
    }
    public String getAddess1(){ 
    return this.addess1;
    }
    public void setAddess1(String addess1){ 
    this.addess1=addess1;
    }
    public String getAddress2(){ 
    return this.address2;
    }
    public void setAddress2(String address2){ 
    this.address2=address2;
    }
    public String getAddress3(){ 
    return this.address3;
    }
    public void setAddress3(String address3){ 
    this.address3=address3;
    }
    public String getEmailid(){ 
    return this.emailid;
    }
    public void setEmailid(String emailid){ 
    this.emailid=emailid;
    }
    public String getZipcode(){ 
    return this.zipcode;
    }
    public void setZipcode(String zipcode){ 
    this.zipcode=zipcode;
    }
    public String getMobileno(){ 
    return this.mobileno;
    }
    public void setMobileno(String mobileno){ 
    this.mobileno=mobileno;
    }
    public Integer getAppusercd(){ 
    return this.appusercd;
    }
    public void setAppusercd(Integer appusercd){ 
    this.appusercd=appusercd;
    }
    public Integer getStatus(){ 
    return this.status;
    }
    public void setStatus(Integer status){ 
    this.status=status;
    }


}

File: frontend/app/src/main/java/com/asuravan/smartsupply/pojo/Shop.kt
package com.asuravan.smartsupply.pojo

import android.os.Parcel
import android.os.Parcelable

class Shop():Parcelable {

     var shopcd: Int? = null
     var shopname: String? = null
     var ownername: String? = null
     var addess1: String? = null
     var address2: String? = null
     var address3: String? = null
     var emailid: String? = null
     var zipcode: String? = null
     var mobileno: String? = null
     var appusercd: Int? = null
     var status: Int? = null

    constructor(parcel: Parcel) : this() {
        shopcd = parcel.readValue(Int::class.java.classLoader) as? Int
        shopname = parcel.readString()
        ownername = parcel.readString()
        addess1 = parcel.readString()
        address2 = parcel.readString()
        address3 = parcel.readString()
        emailid = parcel.readString()
        zipcode = parcel.readString()
        mobileno = parcel.readString()
        appusercd = parcel.readValue(Int::class.java.classLoader) as? Int
        status = parcel.readValue(Int::class.java.classLoader) as? Int
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeValue(shopcd)
        parcel.writeString(shopname)
        parcel.writeString(ownername)
        parcel.writeString(addess1)
        parcel.writeString(address2)
        parcel.writeString(address3)
        parcel.writeString(emailid)
        parcel.writeString(zipcode)
        parcel.writeString(mobileno)
        parcel.writeValue(appusercd)
        parcel.writeValue(status)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Shop> {
        override fun createFromParcel(parcel: Parcel): Shop {
            return Shop(parcel)
        }

        override fun newArray(size: Int): Array<Shop?> {
            return arrayOfNulls(size)
        }
    }


}